src/setup/telemetry/index.ts

```diff
@@ -1,10 +1,23 @@
-import Timing from '@userActions/Timing';
-import CONST from '@src/CONST';
+import * as Sentry from '@sentry/react-native';
+import CONFIG from '@src/CONFIG';
+import pkg from '../../../package.json';
 
-export default function () {
-    Timing.start(CONST.TIMING.SPLASH_SCREEN);
+const navigationIntegration = Sentry.reactNavigationIntegration({
+    enableTimeToInitialDisplay: true,
```

Comment:
can be overwritten using https://docs.sentry.io/platforms/react-native/tracing/instrumentation/time-to-display/#time-to-initial-display-overwrite if we want to await some more UI being loaded (and the default turns out to be falsy)